main.py

```diff
@@ -41,99 +40,175 @@ def check(a, b):
     return new
 
 
-def clean_board():
-    """Returns a new, blank board"""
-    # noinspection PyUnusedLocal
-    return [[Cell() for a in range(config.Height + (2 * config.Cushion))] for b in
-            range(config.Width + 2 * config.Cushion)]
-
-
 def draw(state, a, b, colour):
     """Draws a type of cell (Type) at the desired cell (a,b)"""
-    x = (a - config.Cushion) * config.Size + config.Edge / 2
-    y = (b - config.Cushion) * config.Size + config.Edge / 2
-    s = config.Size - config.Edge
+    x = (a - Board.Cushion) * Board.Size + Board.Edge / 2
+    y = (b - Board.Cushion) * Board.Size + Board.Edge / 2
+    s = Board.Size - Board.Edge
     pygame.draw.rect(Screen, (255, 255, 255), (x, y, s, s))
     if state == config.Square:
         pygame.draw.rect(Screen, colour, (x, y, s, s))
```

Comment:
This is very confusing. The function is drawing the cell not at `a` but at `(a - Board.Cushion) * Board.Size + Board.Edge / 2` and then draws the second point at `Board.Size - Board.Edge` which sounds like the edge of the window. And then if the state `config.Square` it draws the cell a second time, but this time in colour `colour`. Not saying that this must or can be changed, but I'd look into simplifying this. Drawing a cell at (x,y) shouldn't be this hard

Reply:
a and b are the places in the board, x and y are the coordinates on the screen